doc/classes/VisualShape2D.xml

```diff
@@ -0,0 +1,66 @@
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="get_points" qualifiers="const">
+			<return type="PackedVector2Array" />
+			<description>
+				Returns a list of points representing the vertices of the polygon.
+			</description>
+		</method>
+		<method name="get_uvs" qualifiers="const">
+			<return type="PackedVector2Array" />
+			<description>
+				Returns a list of UV values for each vertex of the polygon.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="antialiased" type="bool" setter="set_antialiased" getter="is_antialiased" default="false">
+			If [code]true[/code], the shape will be antialiased. UVs are not supported on antialiased edges. Scaling may affect antialiasing quality.
+			[b]Note:[/b] This may cause unintentional artifacts when using transparency. Consider adding this to a [CanvasGroup] and using its [member CanvasItem.self_modulate] property for transparency. Alternatively, consider using MSAA ([member ProjectSettings.rendering/anti_aliasing/quality/msaa_2d]).
+		</member>
+		<member name="color" type="Color" setter="set_color" getter="get_color" default="Color(1, 1, 1, 1)">
+			The color of the shape.
+		</member>
+		<member name="offset" type="Vector2" setter="set_offset" getter="get_offset" default="Vector2(0, 0)">
+			The offset amount of the shape in pixels. An offset of [code]0,0[/code] will center the shape. This can be used to move the shape without changing the pivot point.
+		</member>
+		<member name="outline_width" type="float" setter="set_outline_width" getter="get_outline_width" default="0.0">
+			If greater than [code]0[/code], the shape uses an outline instead of being filled. This is the outline width of the shape in pixels. UVs are not supported on outlines.
+		</member>
+		<member name="resolution" type="int" setter="set_resolution" getter="get_resolution" default="64">
+			The resolution to use when the [member shape_type] is [constant SHAPE_CIRCLE] or [constant SHAPE_CAPSULE]. The resolution is used as the number of points in the shape. Large values may affect performance.
+		</member>
+		<member name="shape_type" type="int" setter="set_shape_type" getter="get_shape_type" enum="VisualShape2D.ShapeType" default="0">
+			The type of shape to draw.
+		</member>
+		<member name="size" type="Vector2" setter="set_size" getter="get_size" default="Vector2(128, 128)">
+			The size of the shape in pixels.
+		</member>
+	</members>
+	<constants>
+		<constant name="SHAPE_RECTANGLE" value="0" enum="ShapeType">
+			A square or rectangle shape.
+		</constant>
+		<constant name="SHAPE_CIRCLE" value="1" enum="ShapeType">
+			A circle or regular polygon shape. When [member resolution] is high, this approximates a circle. This can be an oval if the [member size] is non-uniform.
```

Comment:
Not too strong on this, feel free to dismiss.
```suggestion
			A circle or regular polygon shape. When [member resolution] is high, this approximates a circle. This can also be an oval if the [member size] is non-uniform.
```